pip/flatpak-pip-generator

```diff
@@ -158,6 +169,16 @@ if opts.requirements_file:
     except FileNotFoundError:
         pass
 
+elif opts.pyproject_file:
```

Comment:
Maybe it should error if both `--requirements-file` and `--pyproject-file` are given. Also please rebase and update `README.md` to include this option.